GVRf/Framework/jni/objects/scene_object_jni.cpp

```diff
@@ -176,7 +179,7 @@ Java_org_gearvrf_NativeSceneObject_attachEyePointeeHolder(
         jlong jeye_pointee_holder) {
     SceneObject* scene_object = reinterpret_cast<SceneObject*>(jscene_object);
     EyePointeeHolder* eye_pointee_holder =
-            reinterpret_cast<EyePointeeHolder*>(jeye_pointee_holder);
+    reinterpret_cast<EyePointeeHolder*>(jeye_pointee_holder);
```

Comment:
I did the Ctrl+Shift+F as per the guideline. This output might be the result of that operation.